consul/src/main/scala/io/buoyant/consul/CatalogNamer.scala

```diff
@@ -51,11 +51,11 @@ class CatalogNamer(
     def get(name: String): DcCache = synchronized {
       activity.sample() match {
         case Activity.Ok(snap) =>
-          snap.get(name).getOrElse {
+          snap.getOrElse(name, {
```

Comment:
FWIW, I tried to make this same change in #247 and @olix0r had [some objections](https://github.com/buoyantio/linkerd/pull/247#discussion_r58918627).